tests/utils.go

```diff
@@ -896,10 +896,10 @@ func AddConfigMapDisk(vmi *v1.VirtualMachineInstance, configMapName string, volu
 }
 
 func AddSecretDisk(vmi *v1.VirtualMachineInstance, secretName string, volumeName string) {
-	AddSecretDiskWithCustomLabel(vmi, secretName, volumeName, "")
+	addSecretDiskWithCustomLabel(vmi, secretName, volumeName, "")
```

Comment:
You are right, but this PR and any of its commit do not try to be perfect, just improve the code (very) little bit. No doubt that it is a small step, but as long as it is in the right direction, I hope it can be accepted as it is.

Reply:
Sure! This can be adjusted in a follow-up PR